core/src/test/scala/org/allenai/common/EnumSpec.scala

```diff
@@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
 
 import spray.json._
 
-class EnumSpec extends UnitSpec {
+import java.io.{ FileOutputStream, FileInputStream }
```

Comment:
I see, I thought you thought the imports were inside the class declaration. I'll fix.